### DISCUSSION TITLE
SerpAPI Key and Skill

### DISCUSSION THREAD

**Discussion by leondape** [StreetLamb/tribe#111] [General]
I've entered a serpapi api key yet there is no skill I could add. What am I doing wrong?

**Comment by StreetLamb** [comment]
Hi @leondape, Serpapi is not supported by default in Tribe. However, it is possible to write a custom tool for it. Alternatively, if you just want to perform a web search, the default search tool in Tribe, DuckDuckGoSearch, should do the job. No keys required to use it.
